addons/survey/static/src/scss/survey_templates_results.scss

```diff
@@ -87,6 +87,14 @@
         border-color: #DEE2E6;
         font-weight: bold;
     }
+    table {
+        font-size: 1rem;
+        &.o_indexed_table {
```

Comment:
Can we go for a more specific class? This makes it look like a generic thing.

Maybe simply "o_survey_results_table" ?

Reply:
Hello @awa-odoo :slightly_smiling_face: 

I made it generic to impact every survey table (not only the indexed one) to avoid any strange flicker (https://tinyurl.com/2fyen8wp) and to also keep a consistency when scrolling through the results :slightly_smiling_face:  It would be weird and would maybe lost the user to have the first question `Where do you live?` with font size 1 rem and when scrolling down having smaller font for the results. 

What do you think? Or do you mean just renaming the class and adding it to every table under the `o_survey_results_question` ? :thinking: This would be a useless specific class though and would add unnecessary classes on the xml tables :thinking: 

What do you think? Do I still make it specific ? :slightly_smiling_face:

Reply:
I was just suggesting renaming "o_indexed_table" to "o_survey_results_table" and renaming accordingly in the xml file yes.

I'm not sure what you mean by "and adding it to every table", you are already adding "o_indexed_table" so we would just rename, nothing more :smile: 

Reasoning is that "o_indexed_table" makes me think that this is a class coming from the /web directory to do some generic styling that we are further customizing in survey, which is not the case, so I prefer something prefixed like other survey classes :slightly_smiling_face:

Reply:
Oh I see ! My bad I misunderstood :sweat_smile: 

Would `o_survey_indexed_table` be ok ? As it isn't any results tables but only the ones with the indexed lines https://tinyurl.com/2llwvqjx

Thanks!

Reply:
Sure, why not, would keep "results" in there though, so it would be "o_survey_results_table_indexed" maybe? A bit long but oh well :slightly_smiling_face: